### PR TITLE
Update to version 4180

### PR DESCRIPTION
**Pull request by andy0130tw**
As title, this patch update the files to Sublime Text v4180 via the provided scripts.

### CODE REVIEW

File: extract-sublime.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ for file in *; do
   if test -f "$file"; then
     # Get its size
     #  `du $file | cut --fields 1`: `4172 plugin_host` -> `4172` (kb)
-    filesize="$(du "$file" | cut --fields 1)"
+    filesize="$(du "$file" | gcut --fields 1)"
```
Comment (twolfson) [inline extract-sublime.sh]: Could you generalize this? I think `gcut` is a macOS-ism. I normally run this script on Linux Mint, which only knows `cut`

Example at the top:
```bash
if $(which cut &> /dev/null); then
  CUT="cut"
elif $(which gcut &> /dev/null); then
  CUT="gcut"
else
  echo "Missing \`cut\` utility. Please install \`cut\` or \`gcut\`" 1>&2
  exit 1
fi

# ...

filesize="$(du "$file" | "$CUT" --fields 1)"
```

**Review by twolfson — CHANGES_REQUESTED** [review]
Generally looks good, thank you! We lose Linux compatibility though for the scripts =/ Can you make those more agnostic? (recommendation added)

Also heads up that you didn't need a "Release ..." commit, that's handled when I run `./release.sh` (also does a tag)

File: extract-sublime.sh (hunk 1 above)
Comment (andy0130tw) [reply extract-sublime.sh]: Sorry that I did run this script on my Mac, and forgot to revert my change to the script. I will do a force push.

Comment (andy0130tw) [reply extract-sublime.sh]: I think you have to reverse the order. BSD's `cut` does not understand the option `--fields`. That was exactly why I installed GNU's `gcut`.

**Review by twolfson — APPROVED** [review]
Looks great! Thanks @andy0130tw!

**Pull request merged**